packages/mux-player/test/player.test.js

```diff
@@ -347,87 +352,129 @@ describe('<mux-player>', () => {
       'https://image.mux.com/bos2bPV3qbFgpVPaQ900Xd5UcdM6WXTmz02WZSz01nJ00tY/storyboard.vtt?token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Ik96VU90ek1nUWhPbkk2MDJ6SlFQbU52THR4MDBnSjJqTlBxN0tTTzAxQlozelEifQ.eyJleHAiOjE5NjE2MDE3NzcsImF1ZCI6InMiLCJzdWIiOiJib3MyYlBWM3FiRmdwVlBhUTkwMFhkNVVjZE02V1hUbXowMldaU3owMW5KMDB0WSJ9.aVd0dsOJUVeQko3BWd9YEhL41Eytf_ZfaBeNzHSSUqU_gREa_jJEVTlRfuiE4g71cKJLSiVTKP7f-F7Txh6DlL8E2SkonfIPB2H0f_3DQxYLso2E8qI4zuJkyxKORbQFLAEB_vSE-2lMbrHXfdpQhv6SrVyu6di9ku0LpFpoyz-_7fVJICr8nhlsqOGt66AYcaa99TXoZ582FWzBaePmWw-WWKYsLvtNjLS9UoxbdVaBRwNylohvhh-i1Y9dNilyNooJ7O8Cj4GuMjeh1pCj0BOrGagxrWrswm3HjUVNUqFq5JCWnJCxgjjwiV4RLZg_4z7gkBXyX7H2-i1dKA3Cpw'
     );
   });
+});
 
-  describe('seek to live behaviors', () => {
-    it('should not have a seek to live button if the stream-type is not live/ll-live', async function () {
-      const playerEl = await fixture(`<mux-player
-        playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
-        stream-type="vod"
-      ></mux-player>`);
-
-      const mediaControllerEl = playerEl.shadowRoot.querySelector('media-controller');
-      const seekToLiveEl = playerEl.shadowRoot.querySelector('.mxp-seek-to-live-button');
-      assert.exists(mediaControllerEl);
-      assert.notExists(seekToLiveEl);
-    });
+describe('<mux-player> playbackId transitions', () => {
```

Comment:
non-blocking: We should probably have tests for most or all props/attrs when 1) changing playbackIds and 2) unsetting a playbackId just to have a clearer definition of done for this effort. That said, I think this is still an improvement, so I don't think it should block the merge.